Documentation/guides/The LiveCode IDE.md

```diff
@@ -586,11 +586,12 @@ placeholder fields which you can navigate with the tab key. Some
 completions may have multiple placeholders with the same name which
 will be edited together. Clicking within a placeholder will select it.
 
-The current automatic completion of control structures has not changed.
+#### Autocomplete Pro
 
-Autocomplete provides the following features:
+> **Note:** Autocomplete is available in LiveCode Indy and above
```

Comment:
Should this say `Autocomplete Pro is available...`?

Reply:
yes it should